### PR TITLE
Fix link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by auchri**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ For more info, see [GitHub Help][help_change_commit_message].
 
 ## Build instructions
 
-See the [readme][README.md#build-instructions] for details on the various build
+See the [README.md](README.md#build-instructions) for details on the various build
```
Comment (robbyoconnor) [inline CONTRIBUTING.md]: That's just the text -- the actual link looks fine...

Comment (robbyoconnor) [inline CONTRIBUTING.md]: That's just the text -- the actual link looks fine...Why is this even a PR?

**Comment by auchri** [comment]
@robbyoconnor No, the link is currently broken:

**Comment by robbyoconnor** [comment]
Your solution didn't fix it [some text here](link here)

**Comment by robbyoconnor** [comment]
Edit nvm -- this message is being deleted.

**Comment by robbyoconnor** [comment]
Hi,

I'd like to add you to my professional network on LinkedIn.

- Robert

Confirm that you know Robert: https://www.linkedin.com/e/v2?e=0-ini67og6-5y&t=ssuw&tracking=eml-guest-invite-cta&ek=invite_guest&sharedKey=8b5_QvU8&invitationId=6130896966575546368








You received an invitation to connect. LinkedIn will use your email address to make suggestions to our members in features like People You May Know. Unsubscribe here: https://www.linkedin.com/e/v2?e=0-ini67og6-5y&t=lun&midToken=AQFmrFVhGEd36Q&ek=invite_guest&loid=AQHlKijhgxIrBwAAAVRVSbcrzLLJKp6z8joMhKj88kp1tp4P19_4Vz113UPeMqA5fsWiJgE8DMdJMgVQuqYX0fSWH1k5FBe1d21Z6cOkfx7Diquf5VB-ehZDVx1batG3m7IX0BHBsUQDqFhMSJZNG_FcjuL75BDI52rox4NRybwnntZmywK2knd29laIWu5aOwEB&eid=0-ini67og6-5y


This email was sent to reply@reply.github.com.

If you need assistance or have questions, please contact LinkedIn Customer Service: https://www.linkedin.com/e/v2?e=0-ini67og6-5y&a=customerServiceUrl&ek=invite_guest

&copy; 2016 LinkedIn Corporation, 2029 Stierlin Court, Mountain View CA 94043. LinkedIn and the LinkedIn logo are registered trademarks of LinkedIn.

**Pull request merged**